src/main/java/com/parkhomovsky/bookstore/model/Book.java

```diff
@@ -40,7 +41,7 @@ public class Book {
     private String description;
     @Column(name = "cover_image")
     private String coverImage;
-    @ManyToMany
+    @ManyToMany(fetch = FetchType.EAGER)
     @JoinTable(
```

Comment:
try to fix it, eager is a bad practice

Reply:
I tried, and finally Bohdan Chupika allowed this